### ISSUE TITLE
How to color important stuff?

### ISSUE THREAD

**Issue by pyromaniac3010** [mr-karan/calert#62]
Is it possible to use colors?
I tried the following config but it did not work as expected:
```
templates:
  message.tmpl: |
    {{- if ne .Status "Resolved" -}}<font color=#FF0000>{{- else -}}<font color=#00FF00>{{- end -}}*({{.Labels.severity | toUpper }}) {{ .Labels.alertname | Title }} - {{.Status | Title }}*</font>
    {{ range .Annotations.SortedPairs -}}
    {{ .Name | Title }}: {{ .Value}}
    {{ end -}}
``` 
This results in this:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/93269573/234026510-ec2dea8b-36c0-4d71-8e86-9f76d0b59fb9.png">

**Comment by flowramps** [comment]
I'm having the same problem and question, has anyone managed to adjust and parameterize it?

**Comment by mr-karan** [comment]
Added documentation in commit 977af35.

**TL;DR**: Google Chat's simple text webhook does **not** support HTML tags like `<font color="...">`. 

For colors and rich formatting, use **CardsV2** templates instead. See the updated [README](https://github.com/mr-karan/calert#cardsv2-support) for an example.

CardsV2 allows structured cards with headers, sections, and styled widgets. Check [Google's CardsV2 documentation](https://developers.google.com/chat/api/reference/rest/v1/cards) for all available options.

**Comment by mr-karan** [comment]
Closing - HTML color tags are not supported by Google Chat's simple text webhook. For colors and rich formatting, use CardsV2 templates instead. Documentation and example added to README.
